doc/sphinx/install.rst

```diff
@@ -8,13 +8,81 @@ Installing Cantera
    :local:
    :depth: 2
 
+.. _sec-install-conda:
+
+Conda
+=====
+
+`Anaconda <https://www.continuum.io/downloads>`_ and `Miniconda
+<http://conda.pydata.org/miniconda.html>`_ are Python distributions for which
+Cantera is available through the `conda` package manager. Both distributions are
+available for Linux, OS X, and Windows. The base Anaconda distribution includes
+a large number of Python packages that are widely used in scientific
+applications. Miniconda is a minimal distribution, where all of the packages
+available in Anaconda (and more) can be installed using the package manager.
+Note that installing Cantera using conda will only provide the Cantera Python
+module. If you want to use the other Cantera interfaces, see the OS-specific
+installation options below.
+
+For more details on how to use conda, see the `conda documentation
+<http://conda.pydata.org/docs/intro.html>`_.
+
+**Option 1: Create a new environment for Cantera**
+
+If you have just installed Anaconda or Miniconda, the following instructions
+will create a conda environment where you can use Cantera. For this example, the
+environment is named *spam*. From the command line, run::
+
+    conda create -n spam -c cantera ipython matplotlib
```

Comment:
This command works most of the time, but sometimes a user can have a altered .condarc file, in which case the command does not search pre-pend http://anaconda.org to the channel name. So to be failsafe, it may be better to suggest this:
`conda create -n conda_env -c cantera cantera ipython matplotlib`